src/strategies/liquidity-token-provide/README.md

```diff
@@ -0,0 +1,31 @@
+# Liquidity Providers
+
+
+Fork from infinityprotocol-liquidity-pools. Fix there is some protocol can't query `users` issue.
+
+This strategy will return the scores of all users who have provided token liquidity on any Uniswap style exchange. Users can change the subGraphURL field to direct their request to a different subgraph.
+
+
+
+
+## Example
+
+The space config will look like this:
+
+```JSON
+{
+  "strategies": [
```

Comment:
Hey can you change this example to have just params object

Reply:
Sure, updated.